src/zope/component/interfaces.py

```diff
@@ -20,33 +20,7 @@
 from zope.interface import Attribute
 from zope.interface import Interface
 
-
-# BBB 2011-09-09, import interfaces from zope.interface
-import zope.deferredimport
-zope.deferredimport.deprecatedFrom(
-    "Import from zope.interface.interfaces",
-    "zope.interface.interfaces",
-    'ComponentLookupError',
-    'Invalid',
-    'IObjectEvent',
-    'ObjectEvent',
-    'IComponentLookup',
-    'IRegistration',
-    'IUtilityRegistration',
-    '_IBaseAdapterRegistration',
-    'IAdapterRegistration',
-    'ISubscriptionAdapterRegistration',
-    'IHandlerRegistration',
-    'IRegistrationEvent',
-    'RegistrationEvent',
-    'IRegistered',
-    'Registered',
-    'IUnregistered',
-    'Unregistered',
-    'IComponentRegistry',
-    'IComponents',
-)
-
+# pylint:disable=inherit-non-class,no-self-argument,no-method-argument
```

Comment:
Do you use local linting? Just wondering about that line. I wouldn't mind adding a canonical linting configuration to `tox.ini`.

Reply:
Yes, I've settled on pylint because it does actual code analysis and doesn't just complain about whitespace.